starter-core/src/main/java/io/micronaut/starter/feature/camunda/Camunda.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 original authors
```

Comment:
I think this should be 2021

Reply:
We use currently 2020 all through the project. This will need to be addressed in a separate PR.